app/uk/gov/hmrc/cgtpropertydisposals/connectors/UpscanConnector.scala

```diff
@@ -67,7 +67,7 @@ class UpscanConnectorImpl @Inject() (playHttpClient: PlayHttpClient, config: Ser
   private val headers: Seq[(String, String)] = Seq(USER_AGENT -> userAgent)
 
   override def downloadFile(upscanSuccess: UpscanSuccess): Future[Either[Error, FileAttachment]] =
-    (upscanSuccess.uploadDetails.get("filename"), upscanSuccess.uploadDetails.get("fileMimeType")) match {
+    (upscanSuccess.uploadDetails.get("fileName"), upscanSuccess.uploadDetails.get("fileMimeType")) match {
```

Comment:
@mikail-khan  this is definitely right isn't it?

Reply:
yes - this is from the upscan readme

```
{
    "reference" : "11370e18-6e24-453e-b45a-76d3e32ea33d",
    "fileStatus" : "READY",
    "downloadUrl" : "https://bucketName.s3.eu-west-2.amazonaws.com?1235676",
    "uploadDetails": {
        "uploadTimestamp": "2018-04-24T09:30:00Z",
        "checksum": "396f101dd52e8b2ace0dcf5ed09b1d1f030e608938510ce46e7a5c7a4e775100",
        "fileName": "test.pdf",
        "fileMimeType": "application/pdf"
    }
}
```

but we now know how reliable that is!

Reply:
let's see then - fingers crossed